app/controllers/tasks_controller.rb

```diff
@@ -4,58 +4,22 @@ class TasksController < ApplicationController
   def index
     tasks = user.tasks
 
-    serialized_tasks = tasks.map do |task|
-      {
-        'id' => task.id,
-        'userId' => user.id,
-        'status' => task.status,
-        'name' => task.name,
-        'createdAt' => task.created_at,
-        'completedAt' => task.completed_at
-      }
-    end
-
-    render(json: serialized_tasks)
+    render(json: tasks.map { |task| TaskSerializer.new(task).as_json })
```

Comment:
você pode atribuir as tasks a uma variavel:
```suggestion
    serialized_tasks = tasks.map { |task| TaskSerializer.new(task).as_json }
    
    render(json: serialized_tasks)
```

Reply:
Yes, sir